### PR TITLE
Data model dec9

### PR DESCRIPTION
**Pull request by trudge-64**
Merging seeddatahelper

### CODE REVIEW

File: Business.Queries/Dtos/JobGroupDto.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Business.Queries.Dtos
```
Comment (ccg-oharan) [inline Business.Queries/Dtos/JobGroupDto.cs:5]: All the DTO's should be in the Business.Dtos project. This allows them to be used in both the Business and Web projects.

File: Business.Queries/JobGroups/GetJobGroupByIdQueryHandler.cs [hunk 2]
```diff
@@ -27,12 +30,9 @@ public Task<JobGroupDto> HandleAsync(GetJobGroupByIdQuery query, CancellationTok
             return _db.JobGroups.Where(e => e.Id == query.Id)
                 .Select(e => new JobGroupDto()
                 {
-                    Id = e.Id,
```
Comment (ccg-oharan) [inline Business.Queries/JobGroups/GetJobGroupByIdQueryHandler.cs:30]: Are the code and name properties supposed to be deleted?

File: Business.Queries/JobPositions/GetAllJobPositionsQueryHandler.cs [hunk 3]
```diff
@@ -23,19 +25,17 @@ public Task<List<JobPositionDto>> HandleAsync(CancellationToken cancellationToke
...
             return _db.JobGroupPositions
                 .Include(e => e.JobGroup)
-                .Include(e=>e.JobGroupLevel)
                 .Include(e => e.JobPosition)
                 .Select(e => new JobPositionDto()
                 {
                     JobGroupId = e.JobGroupId,
                     JobGroupCode = e.JobGroup.Code,
                     JobGroupLevelId = e.JobGroupLevelId,
-                    JobGroupLevelValue = e.JobGroupLevel.LevelValue,
```
Comment (ccg-oharan) [inline Business.Queries/JobPositions/GetAllJobPositionsQueryHandler.cs:33]: Is JobGroupLevelValue supposed to be deleted?

File: Business.Queries/JobPositions/GetJobPositionByIdQueryHandler.cs [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
 using Microsoft.EntityFrameworkCore;
 
 namespace Business.Queries.JobPositions
 {
-        public class GetJobPositionByIdQuery : IQuery<JobPositionDto>
-        {
-            public int Id { get; set; }
-        }
-    public class GetJobPositionByIdQueryHandler : IQueryHandler<GetJobPositionByIdQuery, JobPositionDto>
+    public class GetJobPositionByIdQueryHandler
```
Comment (ccg-oharan) [inline Business.Queries/JobPositions/GetJobPositionByIdQueryHandler.cs:14]: This query handler doesn't implement IQueryHandler. Also, it's named "ById" but doesn't take in an Id parameter. Were the changes to this query handler necessary?

File: DataModel/Configurations/CertificateConfiguration.cs [hunk 5]
```diff
@@ -23,16 +23,16 @@ public void Configure(EntityTypeBuilder<Certificate> builder)
                .HasMaxLength(1000);
 
             builder.Property(b => b.DescEng)
-               .HasMaxLength(1000);
+               .HasMaxLength(500);
 
             builder.Property(b => b.DescFre)
-              .HasMaxLength(1000);
```
Comment (ccg-oharan) [inline DataModel/Configurations/CertificateConfiguration.cs:29]: Can you confirm these fields should be reduced?

File: Web/Pages/Index.cshtml [hunk 6]
```diff
@@ -1,36 +1,21 @@
...
+
+    <li>@Localizer["Welcome"]</li>
+}
+
+<div class="text-center">
+    <h1 class="display-4">@Localizer["Welcome"]</h1>
+    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
 </div>
 
-<div class="container">
```
Comment (ccg-oharan) [inline Web/Pages/Index.cshtml:15]: Is there are reason to revert the page back to the template default text?

File: Web/Pages/Shared/_ThemeGcwu.cshtml [hunk 7]
```diff
@@ -1,20 +1,11 @@
 ﻿@using Microsoft.AspNetCore.Http
 @inject IViewLocalizer Localizer
 @inject IHttpContextAccessor contextAccessor
-@{
+@{ 
 
     var twoLetter = System.Threading.Thread.CurrentThread.CurrentCulture.TwoLetterISOLanguageName;
     var isEnglish = twoLetter == "en";
     var page = contextAccessor.HttpContext.Request.RouteValues["page"];
-    var routedata = new Dictionary<string, string>
```
Comment (ccg-oharan) [inline Web/Pages/Shared/_ThemeGcwu.cshtml:9]: We need the routedata stuff to handle english/french page switching properly.

**Review by ccg-oharan — CHANGES_REQUESTED** [review]
Could you confirm all these changes are correct? It looks like maybe there was an issue with merging conflicts that is reverting changes that Dan put in place.

**Comment by trudge-64** [comment]
Yeah this update was done before the merge this morning.

I will merge and do another pull.

**Pull request closed without merging**